src/service_cmd/runner/runner.go

```diff
@@ -31,14 +31,23 @@ func Run() {
 
 	var perSecondPool redis.Pool
 	if s.RedisPerSecond {
-		perSecondPool = redis.NewPoolImpl(srv.Scope().Scope("redis_per_second_pool"), s.RedisPerSecondSocketType, s.RedisPerSecondUrl, s.RedisPerSecondPoolSize)
+		if s.RedisPerSecondTls {
+			perSecondPool = redis.NewAuthTLSPoolImpl(srv.Scope().Scope("redis_per_second_pool"), s.RedisPerSecondAuth, s.RedisPerSecondUrl, s.RedisPerSecondPoolSize)
+		} else {
+			perSecondPool = redis.NewPoolImpl(srv.Scope().Scope("redis_per_second_pool"), s.RedisSocketType, s.RedisPerSecondUrl, s.RedisPerSecondPoolSize)
+		}
 
 	}
-
+	var otherPool redis.Pool
```

Comment:
Two thoughts here:
1. Should we realistically support enabling TLS for one of the pools but not the other?
2. For the auth case, right now we need the user to set up both the tls envvar and the auth one, should we make it so that if auth is set, tls is set by default?

Reply:
1. This was done this way, since it seemed that the two types of pools where meant to be uniquely configurable. I don't mind having a single tls envvar to enable tls on both pools though. 
2. I think that makes sense, since you'll almost always want tls on when using auth, so having it as the default makes sense with the option to override.

Reply:
1. I guess it is not that much complexity, and it leaves it open for configurability.
2. lgtm, thanks!